### PR TITLE
Add new api for independent disk query by name.

### PR DESCRIPTION
**Pull request by vbauzysvmware**
Ref: https://github.com/terraform-providers/terraform-provider-vcd/issues/73

Add new API to query by disk name. Need to implement terraform independent resource.

### CODE REVIEW

File: govcd/disk.go [hunk 1]
```diff
@@ -380,3 +394,32 @@ func FindDiskByHREF(client *Client, href string) (*Disk, error) {
 	// Return the disk
 	return disk, nil
 }
+
+// Find vm using vApp name and VM name. Returns VMRecord query return type
+func (vdc *Vdc) QueryDisk(diskName string) (DiskRecord, error) {
+
```
Comment (Didainius) [inline govcd/disk.go]: Validate if diskName is not empty as it's done in `func (vdc *Vdc) QueryVM(vappName, vmName string)`

File: govcd/disk.go [hunk 2]
```diff
@@ -380,3 +394,32 @@ func FindDiskByHREF(client *Client, href string) (*Disk, error) {
 	// Return the disk
 	return disk, nil
 }
+
+// Find vm using vApp name and VM name. Returns VMRecord query return type
```
[the remark this answers is not in the document]
Comment (vbauzysvmware) [reply govcd/disk.go]: Fixed

File: govcd/disk.go (hunk 1 above)
Comment (vbauzysvmware) [reply govcd/disk.go]: Fixed

File: govcd/disk.go [hunk 3]
```diff
@@ -30,6 +36,14 @@ func NewDisk(cli *Client) *Disk {
 	}
 }
 
+// create instance with reference to types.DiskRecordType
```
Comment (lvirbalas) [inline govcd/disk.go]: Nit: begin comments with capital letter.

Comment (vbauzysvmware) [reply govcd/disk.go]: Fixed

**Pull request merged**